src/utils/sitemapXml.js

```diff
@@ -0,0 +1,41 @@
+import * as Scrivito from 'scrivito';
+import xml from 'xml';
+import formatDate from './formatDate';
+
+const OBJ_CLASSES_TO_INDEX = [
+  'Author',
+  'Blog',
+  'BlogPost',
+  'Event',
+  'Homepage',
+  'Job',
+  'LandingPage',
```

Comment:
In some websites a "Landing page" is a page outside of the page hierarchy, with no connection to the main website. In these scenarios you don't want landings to be part of the sitemap.

* [ ] Please check if this matches the example app usage of landing pages. If so, we should exclude `LandingPage` from the list.

Reply:
Isn't a `LandingPage` built exactly for the "specialized search via google" use-case? So if should be listed here. If the landing pages is for something else than the google index, then "Should this page be indexed?" can be set to "No" for that landing page.